pages/nft/iscn/_iscnId.vue

```diff
@@ -188,6 +187,11 @@ export default class NFTTestMintPage extends Vue {
     return 'Creating NFT class ...'
   }
 
+  get ogImageArweaveUri(): string {
+    if (!this.ogImageArweaveId) return ''
+    return `ar://${this.ogImageArweaveId}`
+  }
```

Comment:
Can we fallback to `getNftClassImage` if `ogImageArweaveId ` not exists
Also I think there is a bug in the original code, we don't know the class id when creating class. In that case, don't set image